src/Stryker.Core/Stryker.Core.UnitTest/Mutators/StatementMutatorTests.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Shouldly;
+using Stryker.Core.Mutators;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Mutators
+{
+    public class StatementMutatorTests
+    {
+        [Fact]
+        public void ShouldBeMutationlevelStandard()
+        {
+            var target = new StatementMutator();
+            target.MutationLevel.ShouldBe(MutationLevel.Standard);
+        }
+
+        public static IEnumerable<object[]> MutableStatements => new List<object[]>
+        {
+            new object[] { SyntaxFactory.BreakStatement() },
+            new object[] { SyntaxFactory.ContinueStatement() },
+            new object[] { SyntaxFactory.YieldStatement(SyntaxKind.YieldBreakStatement) },
+            new object[]
+            {
+                SyntaxFactory.YieldStatement(
+                    SyntaxKind.YieldReturnStatement,
+                    SyntaxFactory.LiteralExpression(SyntaxKind.NumericLiteralExpression, SyntaxFactory.Literal(1))
+                )
+            },
+            new object[] { SyntaxFactory.ReturnStatement() },
+            new object[] {
+                SyntaxFactory.IfStatement(
+                    SyntaxFactory.LiteralExpression(SyntaxKind.TrueLiteralExpression),
+                    SyntaxFactory.EmptyStatement()
+                )
+            },
+        };
+
+        [Theory]
+        [MemberData(nameof(MutableStatements))]
+        private void ShouldMutate(StatementSyntax statement)
+        {
+            var target = new StatementMutator();
+
+            var result = target.ApplyMutations(statement).ToList();
+
+            var mutation = result.ShouldHaveSingleItem();
+
+            mutation.ReplacementNode.ShouldBeOfType(typeof(EmptyStatementSyntax));
+            mutation.DisplayName.ShouldBe("Statement mutation");
+        }
+
+        [Fact]
+        private void ShouldNotMutate()
+        {
+            SyntaxTree tree = CSharpSyntaxTree.ParseText($@"
+namespace Test
+{{
+    class Program
+    {{
+        delegate void TestDelegate();
+
+        static int Method()
```

Comment:
I'm missing the empty statement?

Reply:
I'm not sure what you mean. Te test it isn't mutated?
After the rework, with a whitelist of mutable statements, I'm not sure if I should add every non-whitelisted statement to the test. What do you think? Every other StatementSyntax is commented in the StatementMutator whitelist